style.css

```diff
@@ -619,25 +579,197 @@ body {
   font-weight: 700;
   font-size: 40px;
   line-height: 52px;
-  margin-left: 50px;
-  margin-bottom: 20px;
 }
 
 .contact-text {
   font-style: normal;
-  font-weight: 400;
   font-size: 20px;
   line-height: 28px;
-  text-align: center;
   color: #ebebff;
-  margin-bottom: 40px;
-}
-
-.contact-form-background-image {
-  position: absolute;
-  width: 233px;
-  height: 670.17px;
-  left: 153px;
-  margin-left: -10px;
-  margin-top: 49px;
+  margin-right: 39px;
+  padding-right: 30px;
+  font-weight: 400;
+  width: 569px;
+  height: 102px;
+}
+
+/* ------------Desktop Version----------- */
+
+@media screen and (min-width: 768px) {
+  .body {
+    width: 90%;
+    background: #fff;
+  }
```

Comment:
There appears to be a white space to the right of the website which is not present in the original design as shown below :point_down: . Kindly check and rectify this anomaly. :pray: 

Your design

![Screenshot from 2022-12-15 22-26-49](https://user-images.githubusercontent.com/65233860/207980708-2ccb6360-db71-4f69-968b-224b5168be23.png)


Figma design

![Screenshot from 2022-12-15 22-27-06](https://user-images.githubusercontent.com/65233860/207980740-b9a7537a-cb52-416b-9231-3299f4502b96.png)